packages/client-sdk-nodejs/src/internal/vector-index-data-client.ts

```diff
@@ -713,7 +728,7 @@ export class VectorIndexDataClient implements IVectorIndexDataClient {
   ): Promise<VectorGetItemBatch.Response> {
     const request = new vectorindex._GetItemBatchRequest({
       index_name: indexName,
-      ids: ids,
+      filter: VectorIndexDataClient.idsToFilterExpression(ids),
```

Comment:
Previously we allowed to select items to delete with a list of ids. We now have an id in set filter expression which is equivalent.